src/posts/asking-why-a-practice-is-not-able-to-accommodate-you.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Asking why a practice is not able to accommodate you
+date: '2021-01-10'
+tags:
+  - ada
+  - oral-deaf
+  - deaf
+  - telemedicine
+---
+
+# Context
+
+## Who was this written by?
+
+This text was written by Mel Chua.
+
+## Who was this written for?
+
+I wrote this text on behalf of myself. I am based in the USA and am a citizen there. I am multiply disabled, and the relevant disability in this situation was my deafness. I use both spoken English and ASL, and had planned on using spoken English with this particular provider. I also knew I could easily look for a different provider if I needed to. If these aspects are different for you, you might need to adapt the text or use a different tactic.
+
+## Who was this written to?
+
+This was written to a prospective practice I was seeking telemedicine services from.
+
+## What prompted this writing?
+
+A few days before my scheduled appointment, I got an email with the following (excerpted):
+
+> ...Unfortunately we are not able to accommodate you in our practice at this time, and for that we sincerely apologize. There are many situations in which we are not setup at this time to provide optimal care...
+
```

Comment:
This gave me a tiny pause because I was expecting a single quote. Are these meant to be two separate quotes? If not, you can add a `>` by itself to join the two paragraphs into one quote.